src/program/snabbvmx/tests/scripts/tcpreplay.lua

```diff
@@ -0,0 +1,87 @@
+local S = require("syscall")
+local Tap = require("apps.tap.tap").Tap
+local lib = require("core.lib")
+local pcap = require("apps.pcap.pcap")
+local pci = require("lib.hardware.pci")
+
+function show_usage (code)
+   print(require("program.tcpreplay.README_inc"))
+   main.exit(code)
+end
+
+function parse_args (args)
+   local handlers = {}
+   local opts = {}
+   function handlers.h ()
+      show_usage(0)
+   end
+   function handlers.D (arg)
+      opts.duration = assert(tonumber(arg), "Duration must be a number")
+   end
+   args = lib.dogetopt(args, handlers, "hD:", { help="h", duration="D" })
+   if #args ~= 2 then show_usage(1) end
+   if not opts.duration then opts.duration = 1 end
+   return opts, unpack(args)
+end
+
+-- TODO: Duplicated function. Move to a common place.
+local function dir_exists (path)
+   local stat = S.stat(path)
+   return stat and stat.isdir
+end
+
+-- TODO: Duplicated function. Move to a common place.
```

Comment:
It's good that there's a TODO, but do we really have to keep adding these duplicated functions? Wouldn't it be better to just remove the duplication once and for all?